motor.cpp

```diff
@@ -23,7 +23,15 @@ void Motor::stop(int motor, int motor2, int motor3, int motor4) {
     }
 }
 
+void Motor::go_all(int speeds[], int directions[]) {
+    for (int i = 0; i < sizeof(speeds) / sizeof(*speeds); i++)
```

Comment:
You can't take the size like that, sadly. I'd recommend declaring number of motors (`MOTORS_NUM` in `motor.h`), also, if motor numbers start from 1, I'd recommend changing the for to: `for (int i = 1; i <= MOTORS_NUM; ++i) Motor::go(i, directions[i-1], speeds[i-1]);